src/app/container/container.component.ts

```diff
@@ -192,6 +198,14 @@ export class ContainerComponent extends Entry implements AfterViewInit {
       this.contactAuthorHREF = this.emailService.composeContactAuthorEmail(this.tool);
       this.requestAccessHREF = this.emailService.composeRequestAccessEmail(this.tool);
       this.sortedVersions = this.getSortedTags(this.tool.workflowVersions, this.defaultVersion);
+      this.entryService.getVerifiedPlatforms(tool.id).subscribe((verifiedVersions: Array<VersionVerifiedPlatform>) => {
+        this.versionsWithVerirfiedPlatforms = verifiedVersions.map(value => Object.assign({}, value));
+      });
+      if (this.selectedVersion) {
+        this.entryService.getVersionsFileTypes(tool.id, this.selectedVersion.id).subscribe((fileTypes: Array<SourceFile.TypeEnum>) => {
```

Comment:
I think you should put this after line 177/183, because it's closer to where selectedVersion changes. And also add another line after 175/181 setting versionsFileTypes to null or empty array.